README.md

```diff
@@ -1,9 +1,79 @@
 
 [![GitHub Super-Linter](https://github.com/actions/typescript-action/actions/workflows/linter.yml/badge.svg)](https://github.com/super-linter/super-linter)
 ![CI](https://github.com/actions/typescript-action/actions/workflows/ci.yml/badge.svg)
 [![Check dist/](https://github.com/actions/typescript-action/actions/workflows/check-dist.yml/badge.svg)](https://github.com/actions/typescript-action/actions/workflows/check-dist.yml)
 [![CodeQL](https://github.com/actions/typescript-action/actions/workflows/codeql-analysis.yml/badge.svg)](https://github.com/actions/typescript-action/actions/workflows/codeql-analysis.yml)
 [![Coverage](./badges/coverage.svg)](./badges/coverage.svg)
 
-This GitHub action will be available in the coming weeks. When available, we will update this README.md file.
+The `run-code-analyzer` GitHub Action scans your code for violations using 
+[Salesforce Code Analyzer](https://forcedotcom.github.io/sfdx-scanner), uploads the results as an artifact, and displays
+the results as a job summary.
+
+# Inputs
+* `run-command` (Default value: `'run'`)
+  * <b>Specifies the Salesforce Code Analyzer command to run.</b><br/>
+    Possible values are: `'run'`, `'run dfa'`.<br/>
+  * For more info on Code Analyzer, read our [documentation](https://forcedotcom.github.io/sfdx-scanner).
+* `run-arguments` (Default value: `'--normalize-severity'`)
+  * <b>Specifies arguments passed to the specified `run-command` value.</b><br/>
+    The arguments provided must include `--normalize-severity`.<br/>
+    To control the output file included in the uploaded artifact, specify an output file name with the `--outfile` argument.<br/>
+    If an output file isn’t specified, results are written to GitHub workflow run logs and a
+    *SalesforceCodeAnalyzerResults.json* file is included in the uploaded artifact.
+  * For a full list of acceptable arguments, read the appropriate Command Reference:
+    * [sf scanner run](https://forcedotcom.github.io/sfdx-scanner/en/v3.x/scanner-commands/run)
+    * [sf scanner run dfa](https://forcedotcom.github.io/sfdx-scanner/en/v3.x/scanner-commands/dfa)
+* `results-artifact-name` (Default: `'code-analyzer-results'`)
+  * <b>Specifies the name of the zip archive job artifact where the results output file is uploaded.</b>
+
+# Outputs
+* `exit-code`
+  * The Salesforce Code Analyzer execution exit code.
+* `num-violations`
+  * The total number of violations found.
+* `num-sev1-violations`
+  * The number of normalized high-severity violations found.
+* `num-sev2-violations`
+  * The number of normalized medium-severity violations found.
+* `num-sev3-violations`
+  * The number of normalized low-severity violations found.
+
+This `run-code-analyzer` action will not exit your GitHub workflow when violations are found. Instead, we recommend adding a
+subsequent step to your workflow that uses these outputs to determine how your workflow should proceed.
+See the example below.
```

Comment:
Delete "See the example below."

Just let the Example section stand on its own.